### ISSUE TITLE
Sequence to sequence with GPT model 

### ISSUE THREAD

**Issue by juliahane** [huggingface/transformers#1587]
Hi, I really appreciate if you could tell me if I can build a seq2seq model with gpt2 like this:
I am getting GPT2 run_generation codes, and I want to finetune it in a way, that I give a 
sequence as a context, and then generate another sequence with gpt2, and then I minimize
the cross-entropy loss between the generated sequence and the expected out, and I want to
modify the run_finetune_lm in a way to do it, I was wondering if this way I can make a seq2seq
model with GPT, thank you.

**Comment by rlouf** [comment]
We are currently working on implementing seq2seq for most models in the library (see https://github.com/huggingface/transformers/pull/1455). I won't be ready before a week or two.

**Comment by rlouf** [comment]
I'm closing this issue, but feel free to reply in #1506 that we leave open for comments on this implementation.
